env/toolshed/group_vars/toolshedservers/vars.yml

```
@@ -36,6 +36,7 @@ galaxy_toolshed_config:
     master: True
     logto: "{{ galaxy_toolshed_log_dir }}/uwsgi.log"
     pythonpath: lib
+    unique-cron: -1 -1 -1 -1 -1 python scripts/tool_shed/build_ts_whoosh_index.py -c config/tool_shed.yml --config-section tool_shed
```

```suggestion
    unique-cron: -5 -1 -1 -1 -1 python scripts/tool_shed/build_ts_whoosh_index.py -c config/tool_shed.yml --config-section tool_shed
```
would be every fifth minute, in case every minute is too expensive

---

This is now running on the MTS, it takes a little over 4 minutes to complete currently.

---

Cool. I'm sure we can speed this up a lot, but that should work for now.

---

I should say, the PR version (every minute) is running. Since it uses `unique-cron` I guess this is ok, it just means the indexes will be constantly rebuilt. It uses one of the four CPUs at 100%, which isn't great.